src/server/node_manager.c

```diff
@@ -3593,13 +3574,10 @@ update2_to_vnode(vnal_t *pvnal, int new, mominfo_t *pmom, int *madenew, int from
 				}
 			}
 		}
-		/* Again, if the vnode has been updated in this cycle, */
-		/* don't reset sharing as it likely was set then       */
-		if ((pnode->nd_modified & NODE_UPDATE_VNL) == 0) {
-			pnode->nd_attr[(int)ND_ATR_Sharing].at_val.at_long = VNS_DFLT_SHARED;
-			pnode->nd_attr[(int)ND_ATR_Sharing].at_flags =
-				(ATR_VFLAG_SET |ATR_VFLAG_DEFLT);
-		}
+	
+		pnode->nd_attr[(int)ND_ATR_Sharing].at_val.at_long = VNS_DFLT_SHARED;
+		pnode->nd_attr[(int)ND_ATR_Sharing].at_flags = (ATR_VFLAG_SET |ATR_VFLAG_DEFLT);
```

Comment:
may not be, actually. Depends on what they wanted when they wrote it. Sometimes they set soemthing to SET | DEFLT but do not set MODCACHE or MODIFY since they may not want to show this on pbsnodes or write to the database. These things are tricky to change. SO i left as it was